hosted_control_planes/hcp_high_availability/hcp-disaster-recovery-oadp.adoc

```diff
@@ -0,0 +1,85 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="hcp-disaster-recovery-oadp"]
+= Disaster recovery for a hosted cluster by using {oadp-short}
+include::_attributes/common-attributes.adoc[]
+:context: hcp-disaster-recovery-oadp
+
+toc::[]
+
+You can use the {oadp-short} Operator to perform disaster recovery on {aws-first} and bare metal.
+
+The disaster recovery process with {oadp-short} involves the following steps:
```

Comment:
You've got `{oadp-short}` here but `{oadp-first}` a few lines later. Maybe swap them so this is the "first'?